fiatconnect-api.md

```diff
@@ -2223,6 +2224,25 @@ platform can be used to consume Fiat Connect services by providing their `mobile
 }
 ```
 
+#### 8.3.2.4. `IfscAccount`
+
+`IfscAccount` is an account schema that represents transfers for `INR` currency.
+
+```
+{
+	accountName: `string`,
+	institutionName: `string`,
+	ifsc: `string`,
```

Comment:
What is a client supposed to provide for the `ifsc` field? It seems like the IFSC number ought to be provided in the `accountNumber` field -- what is the `ifsc` field for?